PlacenoteSDK/PNSDK/LibPlacenote.swift

```diff
@@ -226,6 +231,10 @@ class LibPlacenote {
             libPtr.multiDelegate.onStatusChange(prevStatus: libPtr.prevStatus, currStatus: status)
             libPtr.prevStatus = status
           }
+          libPtr.currStatus = status
+          libPtr.currArkitPose = arkitMat
```

Comment:
We need to move setting of currArkitPose and currMapPose into the if case around the multiDelegate.onPose call above